github/pulls_reviewers.go

```diff
@@ -10,22 +10,31 @@ import (
 	"fmt"
 )
 
-// RequestReviewers creates a review request for the provided GitHub users for the specified pull request.
+// Reviewers represents reviewers of a pull request.
+type Reviewers struct {
+	Users []*User `json:"users,omitempty"`
+	Teams []*Team `json:"teams,omitempty"`
+}
+
+// ReviewersRequest specifies users and teams for a pull request review request.
+type ReviewersRequest struct {
+	Reviewers     []string `json:"reviewers,omitempty"`
+	TeamReviewers []string `json:"team_reviewers,omitempty"`
+}
```

Comment:
Minor, swap the order of these two structs, so `ReviewersRequest`, the input, is on top, and `Reviewers`, the output, is on bottom.